.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
   - EMBER_TRY_SCENARIO=ember-release
   - EMBER_TRY_SCENARIO=ember-beta
   - EMBER_TRY_SCENARIO=ember-canary
+  - EMBER_TRY_SCENARIO=ember-default
```

Comment:
The main benefit I'm seeing is that we wouldn't nave to maintain this list ourselves anymore.